### PR TITLE
Update serde to 1.0

### PR DESCRIPTION
**Pull request by kosta**
Fixes https://github.com/alkis/decimal/issues/30

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,7 @@ bitflags = "~0.7"
 libc = "~0.2"
 ord_subset = { optional = true, version = "~1.2" }
 rustc-serialize = { optional = true, version = "~0.3" }
-serde = { optional = true, version = "~0.9" }
+serde = { optional = true, version = "1.0.0" }
```
Comment (alkis) [inline Cargo.toml]: I think we want 1.0 here.

File: Cargo.toml [hunk 2]
```diff
@@ -23,4 +23,4 @@ default = ["ord_subset", "rustc-serialize", "serde"]
 gcc = "~0.3"
 
 [dev-dependencies]
-serde_json = "~0.9"
+serde_json = "~1.0.0"
```
Comment (alkis) [inline Cargo.toml]: Same here: "1.0"

**Comment by kosta** [comment]
After reading http://doc.crates.io/specifying-dependencies.html#caret-requirements should I simply specify "1"? It has the same effect as "1.0" or "1.0.0": All of these mean >= 1.0.0 < 2.0.0

**Comment by alkis** [comment]
Sounds good to me.

**Comment by kosta** [comment]
Done!

**Comment by alkis** [comment]
Thank you @kosta.

**Pull request merged**